packages/tracing/src/browser/metrics.ts

```diff
@@ -85,6 +93,21 @@ export class MetricsInstrumentation {
             if (tracingInitMarkStartTime === undefined && entry.name === 'sentry-tracing-init') {
               tracingInitMarkStartTime = startTimestamp;
             }
+
+            // capture web vitals
+
+            if (entry.name === 'first-paint') {
+              logger.log('[Measurements] Adding FP (First Paint)');
+              this._measurements['fp'] = { value: entry.startTime };
+              this._measurements['mark.fp'] = { value: startTimestamp };
+            }
```

Comment:
Would we be removing the measure spans in the future? The seem redundant with new measures.

Reply:
We can remove them once we have vertical line markers feature on the span view.